integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -509,6 +509,35 @@ void testAlterTableUpdateColumnComment() {
     checkTableColumns(tableName, updateCommentColumns, getTableInfo(tableName));
   }
 
+  @Test
+  void testAlterTableReplaceColumns() {
+    String tableName = "test_replace_columns_table";
+    dropTableIfExists(tableName);
+
+    createSimpleTable(tableName);
+    List<SparkColumnInfo> simpleTableColumns = getSimpleTableColumn();
+    checkTableColumns(tableName, simpleTableColumns, getTableInfo(tableName));
+
+    //  We must make type compatibility when replace columns to pass Hive IT. Or Will get such
+    // error: Failed to alter Hive table [test_replace_columns_table] in Hive metastore, since Hive
+    // metastore will check the compatibility of column type between the old and new column
+    // positions, please ensure that the type of the new column position is compatible with the old
+    // one, otherwise the alter operation will fail in Hive metastore.
```

Comment:
Could you please add a test to cover this bad case?

Reply:
it's the limitation of hive metastore, could you explain the reason?

Reply:
as I had said it's the limitation of hive metastore, I don't think we need to test it

Reply:
what's the purpose of adding this test?

Reply:
removed the comments to extra concern, and I prefer not to add extra tests to test the underlying capabilities and make test code more complicated.